NuRadioReco/detector/RNO_G/db_mongo_read.py

```diff
@@ -35,7 +35,35 @@ def _impl(self, *method_args, **method_kwargs):
 @six.add_metaclass(NuRadioReco.utilities.metaclasses.Singleton)
 class Database(object):
 
-    def __init__(self, database_connection="env_pw_user", database_name=None):
+    def __init__(self, database_connection="RNOG_public", database_name=None, mongo_kwargs={}):
+        """
+        Interface to the RNO-G hardware database. This class uses the python API `pymongo` for the
+        RNO-G MongoDB.
+
+        This classes allows you to connect to preconfigured mongo clients or select your mongo client freely.
+        The database is accesible with the `self.db` variable.
+
+        Parameters
+        ----------
+
+        database_connection : str (Default: `"RNOG_public"`)
+            Specify mongo client. You have 5 options:
+
+                * `"env_pw_user"`: Connect to a server with the environmental variables
+                `mongo_server`, `mongo_user`, and `mongo_password`
+                * `"RNOG_public"`: Preconfigured connection to read-only RNO-G Hardware DB
+                * `"RNOG_test_public"`: Preconfigured connection to read-only RNO-G Test Hardware DB
+                * `"connection_string"`: Use environmental variable `db_mongo_connection_string` to
+                connect to mongo server
+                * `"mongodb*": Every string which starts with `"mongodb"` will be used to connect to
+                a mongo server
```

Comment:
This indentation isn't right. You shouldn't indent the list as a whole, and the bullet points that span multiple lines should be indented by two spaces. See [here](https://nu-radio.github.io/NuRadioMC/Introduction/pages/contributing.html#lists)

Reply:
Thanks, that was not clear for me by the error msg